script/Arcueid/Day 10/10_00_ARC10_1.txt

```diff
@@ -31,7 +31,7 @@ As I bask in the morning's warm welcome―――
 [sha:6cbbd7b766f75ada376a8dfb0bfea9c54ddc9de8]{
 -- Page 2, Offset 24265.
 -- 無意識のうちに枕元の眼鏡を手に取る。
-I put on the glasses resting on my nightstand,
+I put on my glasses resting on the nightstand,
```

Comment:
This construction feels slightly off to me, as though he could be the one resting on the nightstand. Either revert or maybe adjust to something like
> I pick up my glasses from the nightstand and put them on, my hands...